scope3ai/api/tracer.py

```diff
@@ -7,11 +7,13 @@ class Tracer:
     def __init__(
         self,
         name: str = None,
+        keep_tracers: bool = False,
```

Comment:
Sorry, `keep_traces` not `keep_tracers`, as trace is an artifact of stored in a tracer